src/exception/call_stack.cr

```diff
@@ -19,6 +19,14 @@ end
 
 # :nodoc:
 struct Exception::CallStack
+  # Compute current directory at the beginning so filenames
+  # are always shown relative to the *starting* working directory.
+  CURRENT_DIR = begin
+    dir = Process::INITIAL_PWD
+    dir += File::SEPARATOR unless dir.ends_with?(File::SEPARATOR)
+    dir
+  end
```

Comment:
How us this different from the constant in Process?

Reply:
`Process::INITIAL_PWD` is set to `Dir.current` which doesn't have trailing separator. Here it's being added - btw it's the code which was there already before the changes in #9169.

Reply:
@asterite btw, you committed it yrself, see 43bcde874297feca2b62d0181ce46a0c4c5452c5 :P

Reply:
You probably should have mentioned this is a partial revert of #9169 ;)

Reply:
I meant, if the only difference is an ending separator maybe there's no need to use an entirely new constant for this. But maybe it's fine.